libpod/container.go

```diff
@@ -146,6 +146,8 @@ type ContainerConfig struct {
 	Mounts []string `json:"mounts,omitempty"`
 	// StopSignal is the signal that will be used to stop the container
 	StopSignal uint `json:"stopSignal,omitempty"`
+	// StopTimeout is the signal that will be used to stop the container
+	StopTimeout uint `json:"stopTimeout,omitempty"`
```

Comment:
Should set this to a reasonable default in newContainer

Reply:
Set 10 seconds everywhere off of libpod constant.